frontend/app/components/folder-show.js

```diff
@@ -11,7 +11,7 @@ export default class FolderShowComponent extends Component {
   isFolderEditing = false;
 
   get collapsed() {
-    return this.navService.selectedFolder != this.args.folder;
+    return this.navService.query !== null ? false : this.navService.selectedFolder !== this.args.folder;
```

Comment:
How about `this.navService.selectedFolder !== this.args.folder && this.navService.query === null `?